bazel/repositories.bzl

```diff
@@ -484,7 +484,20 @@ def _com_google_protobuf():
 def _com_github_grpc_grpc():
     _repository_impl("com_github_grpc_grpc")
 
+    if "com_github_nanopb_nanopb" not in native.existing_rules():
```

Comment:
@htuch This is the reason for the previous CI error. I agree it looks kind of hacky but seems to be the best way to include it in Envoy today. Loading `@com_google_grpc_grpc//bazel:grpc_deps.bzl` and call `grpc_deps()` won't work at all in current structure we have in this file. Any better suggestion?

Reply:
If this is a gRPC C core hard dependency, we can add it as an external dep, sure. We do this for other gRPC deps below, e.g. `cares`.